.github/ISSUE_TEMPLATE/blog-post-submission.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Blog Post Submission
+about: Propose new blog post content
+title: "[BLOG]"
+labels: ''
+assignees: BenWilson2
+
+---
+
+## Summary
+
+This template is intended to capture a few base requirements that are needed to be met prior to filing a PR that contains a new blog post submission. 
+
+Please fill out this form in its entirety so that an MLflow maintainer can review and work with you in the process of drafting your blog content and in reviewing your blog submission PR. 
+
+PRs that are filed without a linked Blog Post Submission issue and a subsequent agreement on the content and topics covered for the blog post are not guaranteed to be reviewed or merged.
+
+## Acknowledgements
+
+[ ] I have read through the [contributing guide](https://github.com/mlflow/mlflow-website/blob/main/CONTRIBUTING.md)
+
+[ ] I have configured my local development environment so that I can build a local instance of the MLflow website by following the [development guide](https://github.com/mlflow/mlflow-website/blob/main/DEVELOPMENT_GUIDE.md)
+
+[ ] I have verified that there are no legal considerations associated with the nature of the blog post, its content, or references to organizations, ideas, or individuals contained within my post. If I mention a particular organization, idea, or person, I will provide evidence of consent to post by any organization or individual that is mentioned prior to filing my PR. 
```

Comment:
```suggestion
- [ ] I have read through the [contributing guide](https://github.com/mlflow/mlflow-website/blob/main/CONTRIBUTING.md)

- [ ] I have configured my local development environment so that I can build a local instance of the MLflow website by following the [development guide](https://github.com/mlflow/mlflow-website/blob/main/DEVELOPMENT_GUIDE.md)

- [ ] I have verified that there are no legal considerations associated with the nature of the blog post, its content, or references to organizations, ideas, or individuals contained within my post. If I mention a particular organization, idea, or person, I will provide evidence of consent to post by any organization or individual that is mentioned prior to filing my PR. 
```